code/drasil-lang/lib/Language/Drasil/Expr/Lang.hs

```diff
@@ -109,7 +120,7 @@ data Expr where
   Case     :: Completeness -> [(Expr, Relation)] -> Expr
   -- | Represents a matrix of expressions.
   Matrix   :: [[Expr]] -> Expr
-
+  Set :: Expr -> Expr
```

Comment:
This constructor should take in a `Set` (or list if we don't have an `Ord` instance for `Expr`) of `Expr`s.